src/System.Private.ServiceModel/tools/scripts/WCFCustomSetup.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+Echo Set Up test services
+REM
+
+REM ServiceUri is set, then the service is not local hosted
+if NOT "%ServiceUri%"=="" (
+  goto :done
+)
+
+REM If it's already running, we will not set up/clean up service
+tasklist /FI "imagename eq SelfHostedWCFService.exe" | findstr /i /C:"SelfHostedWCFService.exe" 1>nul
+if [%ERRORLEVEL%]==[0] (
+    goto :PingService
+)
+
+REM run the start wcf service elevated
+start /D %~dp0 RunElevated.vbs StartWCFService.cmd
+Echo WCFServiceCleanupNeeded
+
+
+:PingService
+call powershell -NoProfile -ExecutionPolicy unrestricted %~dp0pingWcfService.ps1 "http://localhost/CrlService.svc/GetCrl?serialNum=b52"
```

Comment:
I feel like the CRL service being co-opted for the PingService feels strange to me...